assets/javascripts/discourse/templates/docs-index.hbs

```diff
@@ -34,6 +34,7 @@
                 <div class="item-controls-buttons">
                   {{d-button class=(if (eq categorySort.type "alpha") "categories-alphabet active" "categories-alphabet") icon=categorySortAlphaIcon action=toggleCategorySort actionParam="alpha"}}
                   {{d-button class=(if (eq categorySort.type "numeric") "categories-amount active" "categories-amount") icon=categorySortNumericIcon action=toggleCategorySort actionParam="numeric"}}
+                  {{plugin-outlet name="categories-controls-buttons-bottom" tagName="" connectorTagName="span"}}
```

Comment:
I'm not sure if `span` is the best choice for most outlets. In almost all cases the default (no tags) is fine:

```suggestion
                  {{plugin-outlet name="categories-controls-buttons-bottom"}}
```